### DISCUSSION TITLE
Incorrect/misleading stats in colour blind themes preview popup

### DISCUSSION THREAD

**Discussion by KDNeufeld** [community/community#15741] [Accessibility] [answered]
The colour blind theme feature is welcome addition, but the text is false/misleading.

In the Feature Preview -> Colour blind themes popup, the text claims "color blindness can affect on average 5-6 % of men".  This is fake news and incorrect stats.  The actual number is more like 8% (1 in 12 men).

There are many articles that show it's 8%, not 5%.

https://en.wikipedia.org/wiki/Color_blindness
https://www.healthline.com/health/deuteranopia#:~:text=Research%2C%20including%20a%202018%20study,the%20UK%20National%20Health%20Service.

**Comment by cehfisher** [comment]
Hello @KDNeufeld - thank you for providing this feedback! We were unable to locate the popup you mentioned. It might be possible that this popup was only temporary, but we wanted to follow-up and ask if you had a URL you could provide? Thank you!

**Reply by KDNeufeld** [reply]
Hello @cehfisher, this is popup that I was referencing. Under your GitHub user profile icon, select **Feature Preview**.  

I use this theme all the time and love it, so many thanks for this.

I was just point out that the description is misleading and incorrect. Most articles show the stats to be more like 1 in 12 men (8%) and 1 in 200 women (0.5%). i.e. [here](https://www.express-scripts.ca/raising-health/colour-blindness-what-colours-do-you-see#:~:text=An%20estimated%202.6%20million%20Canadians,gene%20on%20their%20X%20chromosome.), [here](https://opto.ca/eye-health-library/colour-deficiency), [here](https://www.colorblindguide.com/post/colorblind-people-population-live-counter), and [here](https://www.colourblindawareness.org/colour-blindness/).

![image](https://github.com/community/community/assets/55215368/477ec15d-dc76-4d2e-b67a-9149c99c002a)

![image](https://github.com/community/community/assets/55215368/666dee25-4617-46d2-b848-b5d2ff2eac61)
I think I clicked the Give feedback link on the bottom.

**Reply by cehfisher** [reply]
Thank you so much @KDNeufeld for that additional context. We can replicate this and understand the statistics need to be modified. We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Great news @KDNeufeld - this issue has been resolved! We made the changes you suggested and also updated the stats for women which incorrectly read 0.4% instead of 0.5% 🎉 

<img width="958" alt="Screenshot showing updated statistics for colorblindness." src="https://github.com/community/community/assets/15240149/de94a404-b0bf-4106-bb53-26e65a29f682">

**Reply by KDNeufeld** [reply]
That's great! Thanks!
